src/components/Auth.vue

```diff
@@ -19,50 +21,94 @@
       </div>
     </div>
   </form>
+  <form>
+    <label>UDT Count</label>
+    <input type="number" v-model="count" />
+    <a-button type="primary" @click.prevent="issue()">Issue</a-button>
+  </form>
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue'
-import Rpc from './rpc'
-import Utils from './utils'
+import { defineComponent } from "vue"
+import Rpc from "../utils/rpc"
+import Utils from "../utils/utils"
+import Const from "../utils/const"
+import BN from "bn.js"
 
 interface Wallet {
+  lockScript: {};
   address: string;
-  free: string | undefined;
-  capacity: string | undefined;
+  free: string;
+  capacity: string;
+  lockHash: string;
+}
+interface Summary {
+  free: number;
+  capacity: number;
+}
+interface Transaction {
+  version: string;
+  cellDeps: Array<object>;
+  headerDeps: Array<object>;
+  inputs: Array<object>;
+  outputs: Array<object>;
+  witnesses: Array<string | object>;
+  outputsData: Array<string>;
+}
+interface Cell {
+  output_data: string;
```

Comment:
Why not use `camelCase` for `Cell`?

Reply:
I found example in type script document that the first letter were  a capital.
https://www.tslang.cn/docs/handbook/interfaces.html

Reply:
I mean `output_data`, not `Cell`.